mumblepahelper.h

```diff
@@ -61,6 +62,12 @@ public slots:
     void on_action_Unlink_triggered(bool);
     void on_action_Quit_triggered(bool);
     void on_action_Rescan_triggered(bool);
+
+    void on_action_UserPluginsLocation_triggered(bool);
+    void on_action_SystemPluginsLocation_triggered(bool);
+	void on_CurrentPlugins_clicked(bool);
```

Comment:
You should probably use http://doc.qt.io/qt-4.8/qcheckbox.html#stateChanged instead of clicked.